### DISCUSSION TITLE
Is Vue too complex?

### DISCUSSION THREAD

**Discussion by 0x00A1FA** [vuejs/language-tools#2351] [General]
Hi,

I have this feeling that Vue is getting more and more complex and I think that the complexity of Volar is proof for that. Is it just me, or wouldn't it be great to just use JSX instead, like React?

I like React/JSX because you don't need any extensions and everything works outside the box.

Looking forward to a constructive dialogue.

Regards,
Al

**Comment by johnsoncodehk** [comment]
This repo only focuses on the tooling implementation of Vue. Please submit the design discussion to the Vue community.
